google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ChangeStreamMutation.java

```diff
@@ -110,8 +109,7 @@ static Builder createGcMutation(
   @Nonnull
   public abstract String getSourceClusterId();
 
-  /** Get the commit timestamp of the current mutation. */
-  public abstract Instant getCommitTimestamp();
+  public abstract java.time.Instant getCommitTimestamp();
```

Comment:
We still need the threeten getter here, since this is used by beam users. Same for Heartbeat below